### DISCUSSION TITLE
Grove - Vision AI Module | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#388] [Q&A]
# Grove - Vision AI Module | Seeed Studio Wiki

Grove - Vision AI Module

https://wiki.seeedstudio.com/Grove-Vision-AI-Module/#update-bl702-chip-firmware

**Comment by Abhi83166** [comment]
Whenever i m trying to paste the .uf2, around 70 or 80 % progress it closes automatically and error pops up mentioning An unexpected error is keeping you from copying the file......................
error code 0x800701B1:- A device which does not exist was specified

**Reply by LynnL4** [reply]
You can avoid this issue by using a terminal
`cp xxx.uf2 <path to vision ai>`
